data/internal_id.dat

```diff
@@ -0,0 +1,123 @@
+[tech]
+0x01010000 = Autonomous Vehicles
+0x01010001 = Sociology
+0x01010002 = Voice Synthesis
+0x01010003 = Simulacra
+0x01010004 = Lunar Rocketry
+0x01010005 = Stealth
+0x01010006 = Advanced Intrusion
+0x01010007 = Space-Time Manipulation
+0x01010008 = Leech Satellite
+0x01010009 = Advanced Arbitrage
+0x0101000a = Advanced Stealth
+0x0101000b = Parallel Computation
+0x0101000c = Microchip Design
+0x0101000d = Advanced Microchip Design
+0x0101000e = Quantum Computing
+0x0101000f = Quantum Entanglement
+0x01010010 = Autonomous Computing
+0x01010011 = Advanced Quantum Computing
+0x01010012 = Exploit Discovery/Repair
+0x01010013 = Telepresence
+0x01010014 = Advanced Memetics
+0x01010015 = Media Manipulation
+0x01010016 = Advanced Database Manipulation
+0x01010017 = Internet Traffic Manipulation
+0x01010018 = Memetics
+0x01010019 = Fusion Rocketry
+0x0101001a = Advanced Autonomous Vehicles
+0x0101001b = Intrusion
+0x0101001c = Stock Manipulation
+0x0101001d = Fusion Reactor | Fusion Power
```

Comment:
I'm not excited about this being an alias list due to the possibility of collisions after renames.